GTBitsOfGood/canine-assistants
# Daniel mark logs resolved [FE] #135

## [FE] Mark Logs As Resolved

Issue Number(s): #135

What does this PR change and why?

adds frontend functionality to each pages/dog/[id]. Mainly chaning Log component and Tabs, do display red dot, Resolved/Resolve icons, adding ResolveLogModal and ResolvedLogModal, and integrating acess levels for different user roles (fetching user by userid instead of using NextAuth).

### Checklist

- [ Yes ] Requirements have been implemented
- [ Yes ] Acceptance criteria is met
- [ Yes ] Database schema [docs](https://www.notion.so/gtbitsofgood/Database-Schema-Docs-fd377ded10b54890b65fd99025488cee?pvs=4) have been updated or are not necessary
- [ Yes ] Relevant reviewers (EM) have been assigned to this PR

### Critical Changes

- Database change / migration to run
- Environment config change
- Breaking API change

### Related PRs

PR Number(s): N/A

### How To Test

Set role in website to either admin or manager, and test the functionality by doing to an individual dog (who has a log already) and resolving, editing, or deleting. Also, make your own logs and verify that access levels and functionalities are correct. Please let me know if there are any FE tweaks or bugs I should patch up! :)
